lib/diagnostic.rb

```diff
@@ -45,7 +59,14 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+require_relative 'Person'
+
+class Developer < Person
+  def solve_problems
+    puts "think think think"
+  end
+end
+
 ##
```

Comment:
Great! 
Pay close attention to the language in the question. To return the string we don't need `puts` there
That will actually print the string vs returning it.